### DISCUSSION TITLE
Autocorrelation problems

### DISCUSSION THREAD

**Discussion by AlvicePC** [ROBelgium/MSNoise#284] [Q&A] [answered]
Hi, I did cc for pairs of stations and I'm trying to make it for a single station but is not working, when the comput_cc command finish i only see the ZZ (default component for pairs of station) in the folder for the current filter.
I only mark one station to be used in configuration of stations.
In the component_to_compute_for_single_station I have ZZ,ZE,ZN.
In the cc_type for Autocorrelation i have CC
and cc_type for SC I have CC too, should I change those parameters? I search in the documentation the answer but there's no an specification.

**Comment by ThomasLecocq** [comment, marked as answer]
Try setting ZZ,EZ,NZ , alphabetically sorted, that might be the issue

**Reply by AlvicePC** [reply]
I did it, still not showing folders for the single station component, but I can see how the time of process increase with the configuration for single station and when I run the compute_cc it shows in the message that it will compute components [ EZ, NZ, ZZ ] for single station, but in every process say "New CC job........ (10 pairs with 5 stations). in other discussion you told me that I have to run the New_jobs command after change the parameters and I did it but it didn't found new jobs.

**Reply by ThomasLecocq** [reply]
Did you scan the Horizontal components too? They are in the data_availability table?

**Reply by AlvicePC** [reply]
yes, I have Z, E, and N component scanned. Here you can see. 
![16680250624445242529244548758621](https://user-images.githubusercontent.com/86736263/200933163-05f52e0b-260b-4e15-a592-30567d256596.jpg)

**Reply by ThomasLecocq** [reply]
What does the code say when running with the -v verbose mode? Are the traces read? 

Which version are you using?

**Reply by AlvicePC** [reply]
I don't know how to run in verbose mode.  I'm using Msnoise version 1.6.2

**Reply by ThomasLecocq** [reply]
Use the -v in front of the command, as documented

**Reply by AlvicePC** [reply]
got it. 
![16680290881891319077477853041139](https://user-images.githubusercontent.com/86736263/200945057-09146af8-c19b-42a4-a4db-c6cf02e65d7f.jpg)

**Reply by AlvicePC** [reply]
THOMAS! I think that I know what I was doing wrong, at the time to run new_jobs my the DA was all marked in A, then the command don't get new jobs, but I mark all like M and then it took more jobs now, it pass from 3200 to 4700, I will run compute_cc again and I will give you an update of my situation. Thanks al lot for helping me.

**Reply by AlvicePC** [reply]
ITS ACTUALLY SHOWING AUTOCCORELATED FOLDERS 
![IMG-20221109-WA0009](https://user-images.githubusercontent.com/86736263/200956371-234439de-e55c-48ed-8a98-68136750a66a.jpeg)
